### PR TITLE
A couple of plugin patches

### LINKED ISSUE #1230: [Bug] anonymiseFileNames plugin wrongly scrambles .tar.[gx]z extension
### Discord Account

0x150#9699

### What happens when the bug or crash occurs?

Uploading a file ending in ".tar.[gx]z" results in the filename being scrambled to "(random file name).[gx]z", nuking the `.tar` extension part.

### What is the expected behaviour?

The plugin should recognize the ".tar.[gx]z" suffix, and include it as one extension

### How do you recreate this bug or crash?

1. Upload a file called "test.tar.gz"
2. Observe, the file name is changed to "(randomized).gz"

### Errors

\-

### Request Agreement

- [X] I am using Discord Stable or tried on Stable and this bug happens there as well

### PR DESCRIPTION
**Pull request by KronsyC**
Patched a visual issue with the membercount plugin and some themes colliding

also resolved #1230 by adding support for the common `.tar.??` compound file extensions (more can be trivially added by updating an array)

### CODE REVIEW

**Comment by KronsyC** [comment]
Sorry, Accidentally left some stuff from dev in here, will reopen

**Comment by KronsyC** [comment]
Added another fix which resolves #1186

**Comment by KronsyC** [comment]
Is the accidental newline i added to the pnpm file breaking stuff?

**Comment by Vendicated** [comment]
please follow our naming convention and fix lint

**Comment by KronsyC** [comment]
alrighty, on it

**Comment by KronsyC** [comment]
Should be passing now

**Comment by KronsyC** [comment]
Fixed merge conflict

File: src/plugins/memberCount.tsx [hunk 1]
```diff
@@ -52,11 +52,11 @@ function MemberCount() {
     return (
         <Flex id="vc-membercount" style={{
             marginTop: "1em",
-            marginBottom: "-.5em",
             paddingInline: "1em",
             justifyContent: "center",
             alignContent: "center",
-            gap: 0
+            gap: 0,
+            zIndex: 100,
```
Comment (Vendicated) [inline src/plugins/memberCount.tsx:59]: what is this for?

Comment (KronsyC) [reply src/plugins/memberCount.tsx:59]: Didn't mean to leave that in, that was my initial attempt of pushing the member count above the member list

**Comment by Vendicated** [comment]
Superseded by https://github.com/Vendicated/Vencord/commit/fe80b8cc8586f3119076a99c610f36d7824985a9

**Comment by Vendicated** [comment]
thank you anyway!

**Pull request closed without merging**